en/templating.texy

```diff
@@ -368,6 +368,18 @@ $latte->render('template.latte', $parameters);
 // or $html = $latte->renderToString('template.latte', $parameters);
 \--
 
+
+Easier template usage in presenter or components
+================================================
+
+If we want, for example, to send a [pretty formatted email | mailing#templates], we can use current presenter and its method `createTemplate()` instead of using `Latte\Engine`. It will register all the necessary macros and filters for us.
+
+/--php
+$template = $this->createTemplate();
+$template->setFile('template.latte');
```

Comment:
možná bych tu naznačil, že se jedná o absolutní a ne relativní cestu, jak to teďka vypadá.